src/api/app/models/concerns/staging_project.rb

```diff
@@ -155,6 +182,7 @@ def cache_problems
   end
 
   def state
+    return :accepting if Delayed::Job.where("handler LIKE '%job_class: StagingMergeProjectJob% project_id: #{id}%'").exists?
```

Comment:
How can this be true if you use perform_now? 

> perform_now() Link
Performs the job immediately. The job is not sent to the queueing adapter but directly executed by blocking the execution of others until it's finished.

Reply:
Alright. I'll change it back to perfom_later. We can later than change the picked queue or  prioritize this differently.